src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  padding: 0 16px;
+
+  &__star {
+    width: 16px;
+    height: 15px;
+    background-image: url('../images/star.svg');
+    background-size: contain;
+  }
+
+  &__reviews-text {
+    padding-left: 17px;
+    font-size: 10px;
+    color: $card-title-color;
```

Comment:
Variables should be defined in a separate file according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Ensure `$card-title-color` is defined in `variables.scss`.